CITATIONS.md

```diff
@@ -62,6 +62,8 @@
 
 - [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
 
+  > Andrews, S. (2010). FastQC: A Quality Control Tool for High Throughput Sequence Data [Online]. Available online https://www.bioinformatics.babraham.ac.uk/projects/fastqc/.
```

Comment:
I'll update TEMPLATE to be up to date on that

Reply:
Perhaps the format of the citation is like that because it is "just" citing a homepage, and not a journal publication? 🤔
 
Just removing the brackets around the year seems to lead to a somewhat unconventional format of the citation. On this [homepage](https://manual.omicsbox.biobam.com/user-manual/general-tools/fastq-tools/fastq-quality-check/) it says to cite that online publication like we have done.

Reply:
Aha, it make sense